internal/app/server/server.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Envoyproxy Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package server
+
+import (
+	"context"
+	"net"
+
+	"github.com/envoyproxy/xds-relay/internal/pkg/log"
+
+	api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
use `v3` instead?

Reply:
Chatted offline. go-control-plane isn't in a state where it supports v3 APIs. @jyotimahapatra will open an issue and link it here.